Ginger/Ginger/Actions/ActionConversion/SelectBusinessFlowWzardPage.xaml.cs

```diff
@@ -0,0 +1,188 @@
+#region License
+/*
+Copyright © 2014-2019 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Common.Enums;
+using Amdocs.Ginger.Repository;
+using Ginger.SolutionWindows.TreeViewItems;
+using Ginger.SolutionWindows.TreeViewItems.ApplicationModelsTreeItems;
+using Ginger.UserControls;
+using GingerCore;
+using GingerWPF.UserControlsLib.UCTreeView;
+using GingerWPF.WizardLib;
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Controls;
+
+namespace Ginger.Actions.ActionConversion
+{
+    /// <summary>
+    /// Interaction logic for SelectBusinessFlowWzardPage.xaml
+    /// </summary>
+    public partial class SelectBusinessFlowWzardPage : Page, IWizardPage
+    {
+        ActionsConversionWizard mWizard;
+        public object BusinessFlowFolder { get; set; }
+        SingleItemTreeViewSelectionPage mBFSelectionPage = null;
+        ObservableList<BusinessFlow> ListOfBusinessFlow = null;
+        Context mContext = null;
+
+        public SelectBusinessFlowWzardPage(ObservableList<BusinessFlow> listOfBusinessFlow, Context context)
+        {
+            InitializeComponent();
+
+            ListOfBusinessFlow = listOfBusinessFlow;
+            mContext = context;
+        }
+
+        public void WizardEvent(WizardEventArgs WizardEventArgs)
+        {
+            switch (WizardEventArgs.EventType)
+            {
+                case EventType.Init:
+                    mWizard = (ActionsConversionWizard)WizardEventArgs.Wizard;
+                    SetGridsView();
+                    break;
+                case EventType.LeavingForNextPage:
+                    SetActivitiesSelected();
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// This method is used to set the acitvities for selected to conversion
+        /// </summary>
+        private void SetActivitiesSelected()
+        {
+            foreach (var businessFlow in ListOfBusinessFlow)
+            {
+                if (businessFlow.Selected)
```

Comment:
for what you using businessFlow.Selected if we using BusinessFlowToConvert class?

Reply:
as above removed